src/server/controllers/studentsController.js

```diff
@@ -0,0 +1,6 @@
+const getAllStudents = require('../helpers/studentsHelper');
+
+exports.getStudents = async (req, res) => {
+  const students = await getAllStudents();
```

Comment:
I can imagine having all three of these functions with appropriate endpoints for each:
- getAllStudents();
- getCurrentStudents();  (would need to review the cohort list for all current cohorts to see which are active
- getCohortStudents(cohortID);

Reply:
yeah! Planning to add those next